entrypoint.sh

```diff
@@ -79,8 +79,14 @@ elif [ $INPUT_ACTION = "validate-inputs" ]; then
 
     EXITCODE=0
 
-    echo "sprocket validate-inputs --inputs $INPUT_INPUTS_FILE $INPUT_WDL_FILE"
-    sprocket validate-inputs --inputs $INPUT_INPUTS_FILE $INPUT_WDL_FILE || EXITCODE=$(($? || EXITCODE))
+    IFS=', ' read -r -a input_files <<< "$INPUT_INPUTS_FILE"
+    IFS=', ' read -r -a wdl_files <<< "$INPUT_WDL_FILE"
```

Comment:
```suggestion
    IFS=',' read -r -a input_files <<< "$INPUT_INPUTS_FILE"
    IFS=',' read -r -a wdl_files <<< "$INPUT_WDL_FILE"
```
Asking you without digging into the `man` page: would this suggestion accommodate either with or without the space in the delimiter? Probably depends on the quoting further down.

Reply:
Your question made me notice that I'd forgot to save the file with my explanation comments. I've rectified that and pushed. `IFS` treats every character specified as a potential separator. The `bash` default is ` \t\n`. So I'm allowing both `,` and ` ` to be used here to separate the fields. I'd be OK restricting to just `,`.

Reply:
Oops. Yea I misunderstood how `IFS` works. I think the more user-proof implementation here is going to be just `,`. As stated in my other comment, I would expect no one to be using spaces in their paths, but... users can't be trusted 😅